controllers/replace_misconfigured_pods.go

```diff
@@ -78,22 +64,11 @@ func (c ReplaceMisconfiguredPods) Reconcile(r *FoundationDBClusterReconciler, co
 			continue
 		}
 
-		_, idNum, err := ParseInstanceID(instanceID)
+		hasNewRemoval, err := instanceNeedsRemovalForPVC(cluster, pvc)
```

Comment:
```suggestion
		needsNewRemoval, err := instanceNeedsRemovalForPVC(cluster, pvc)
```

we already have `hasNewRemovals` the trailing `s` can easily be overlooked.